packages/@angular/cli/tasks/eject.ts

```diff
@@ -434,6 +436,13 @@ export default Task.extend({
             Your package.json scripts needs to not contain a start script as it will be overwritten.
           `);
         }
+        if (scripts['pree2e'] && scripts['pree2e'] !== pree2eNpmScript && !force) {
+          // tslint:disable-next-line:max-line-length
```

Comment:
You're using `oneLine` so you can put your string in multiple lines and they'll be joined. Remove this tslint exception.